SPEC.md

```diff
@@ -2601,6 +2602,20 @@ Integer ylen = length(ys) # 3
 Integer zlen = length(zs) # 0
 ```
 
+## Array[X] flatten(Array[Array[X]])
+
+Given an array of arrays, the `flatten` function concatentes all the
```

Comment:
I'm assuming this won't de-dup, can you make that clear? Or note that it does if I'm wrong.

Reply:
I agree - "No du-dup" is the correct answer IMO (since they're arrays, not sets)